ocaml-eglot.el

```diff
@@ -294,14 +302,14 @@ If there is no available holes, it returns the first one of HOLES."
 
 (defun ocaml-eglot--first-hole-in (start end)
   "Jump to the first hole in a given range denoted by START and END."
-  (when-let ((hole (ocaml-eglot--get-first-hole-in start end))
+  (when-let* ((hole (ocaml-eglot--get-first-hole-in start end))
```

Comment:
The line below is not properly indented.

Reply:
I run `indent-buffer` on https://github.com/tarides/ocaml-eglot/pull/37/commits/7f8ab06543f689143dc957e24660e4bd3f9c4a10